docs/sources/setup-grafana/configure-security/configure-authentication/gitlab/index.md

```diff
@@ -81,6 +81,18 @@ to login on your Grafana instance.
 You can limit access to only members of a given group or list of
 groups by setting the `allowed_groups` option.
 
+### Access token expiration check and configure refresh token
```

Comment:
Adjust wording in this section as suggested above.

Reply:
Done, thanks!